scripts/konflux/version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# This script is used by the Konflux dockerfile to get the correct version to
+# compile in to the binary.  If HEAD points to a git tag, use that; otherwise
+# use `git describe ...`.
+
```

Comment:
[nit]
```suggestion

set -euo pipefail

```

mainly for consistency with other scripts.